### ISSUE TITLE
Wrong conversion from RGB uncompressed

### ISSUE THREAD

**Issue by Joet73** [rigaya/QSVEnc#206]
When I encode a RGB24 uncompressed file using, the result is somewhat inverted.
Command line is
`QSVEncC64  --avi --cbr 15000 --codec hevc --quality best -i "Input_RGB24.avi" -o "Output.mp4"`

Original:
![image](https://github.com/rigaya/QSVEnc/assets/127707496/4f1e86bc-4b5e-40f5-b4ba-8c387e91502e)

Render:
![image](https://github.com/rigaya/QSVEnc/assets/127707496/ca7d30c6-7786-4f15-af31-f6db758cb58d)

MediaInfo of the original file
```
Video
ID                                       : 0
Format                                   : RGB
Codec ID                                 : 0x00000000
Codec ID/Info                            : Basic Windows bitmap format. 1, 4 and 8 bpp versions are palettised. 16, 24 and 32bpp contain raw RGB samples
Duration                                 : 10 s 0 ms
Bit rate                                 : 2 488 Mb/s
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate                               : 50.000 FPS
Color space                              : RGB
Bit depth                                : 8 bits
Bits/(Pixel*Frame)                       : 24.000
Stream size                              : 2.90 GiB (100%)
```

**Comment by rigaya** [comment]
Thank you for letting me know the issue, shall be fixed with [QSVEnc 7.67](https://github.com/rigaya/QSVEnc/releases/tag/7.67).

**Comment by Joet73** [comment]
Thank you @rigaya .
The latest version fixes that issue.
